templates/core/Cargo.toml.example

```diff
@@ -1,4 +1,4 @@
-{# The example Cargo.toml used in about.html #}
+{~# The example Cargo.toml used in about.html #~}
```

Comment:
Is this not being rendered by tera, not handlebars? AFAICT from the docs tera has no way to control whitespace in comments (though, if it did, it'd be `{#- The example ... -#}`).

Reply:
Yeah, this will be rendered inline, and `-#}` doesn't work to suppress the post-comment-whitespace:

```
"{# foobar #}\n"
=> Ok("\n")

"{# foobar -#}\n"
=> Ok("\n")

"{~# foobar #~}\n"
=> Ok("{~# foobar #~}\n")

"{% if true %}{% endif -%}\n"
=> Ok("")
```

Reply:
When I check the `/about/metadata`, this actually makes the comment visible in the page, which I assume is not what we want.

Reply:
we could fix this here to reduce the space on the top of the code-section on that page, or revert this change

Reply:
Ok, fixed it by putting `[package]` on the same line as the comment.